js/flashchord.js

```diff
@@ -3,14 +3,14 @@
 const AudioContext = window.AudioContext || window.webkitAudioContext;
 const audioCtx = new AudioContext();
 
-var $intervalId;
-var $tempo = 100;
-var $bars = 1;
-var $chord;
-var $next_chord;
-var $flash_chord_running = false;
-
-$(document).ready(function() {
+let $intervalId;
```

Comment:
These shouldn't be declared with `let` as they are outside the block context and in the global context. These should remain `var` declared.

Reply:
it's my understanding that var is no longer used after es6. 
This [article](https://dev.to/johnwolfe820/should-you-never-truly-use-var-bdi) appears to be a discussion of this var vs let 
I will do some more digging and I am less adamant on this than I am on the #13 issue above.  Cheers!

Reply:
[From this page](https://www.w3schools.com/js/js_let.asp), it sounds like the difference between `var` and `let` is that `let` allows you to redeclare a global variable with local scope and `var` does not.

> Redeclaring a let variable with var, in the same scope, or in the same block, is not allowed:
``` javascript
let x = 2;       // Allowed
var x = 3;       // Not allowed

{
  let x = 4;   // Allowed
  var x = 5;   // Not allowed
}
```
If that's the only difference, then I think we would want to stick with `var` for global non-`const` variables. Everything else within the scope of a function or loop/conditional structure would use `let`. What do you think?